django/test/runner.py

```diff
@@ -438,6 +439,7 @@ def _init_worker(
             process_setup(*process_setup_args)
         django.setup()
         setup_test_environment(debug=debug_mode)
+        call_command("check", verbosity=verbosity, databases=used_aliases)
```

Comment:
Thanks. When running in parallel and multiprocessing method is "spawn", can we skip the call to `run_checks()` in DiscoverRunner? It seems wasteful to do it in both places. (And add a test case asserting against that.)

Reply:
Just tried skipping `run_checks()` in DiscoverRunner, and then found the parallel test runner doesn't exit cleanly if it encounters a system check error.

Reply:
Given my comment below about de-duping the results, perhaps it's easier to just leave the run_checks() call in discover runner alone, and then just run the checks in the parallel runner with verbosity=0?

Reply:
Done in 7ee8e7bbfd!